internal/distributed/proxy/service.go

```diff
@@ -90,7 +85,7 @@ func NewServer(ctx context.Context, factory msgstream.Factory) (*Server, error)
 }
 
 func (s *Server) startGrpcLoop(grpcPort int) {
-
+	s.wg.Add(1)
```

Comment:
little risk here.
If external  execute `go startGrpc()` and then immediately  call `wg.wait`

Maybe startGrpc hasn't added(1) before wg.wait succeeded

Reply:
s.wg.Add(1) should execute before startGrpcLoop

Reply:
OK, got it.